From 3fbb596e53524e78703b76c4fdc33cd6ac62f777 Mon Sep 17 00:00:00 2001
From: Madhurkiran Harikrishnan <madhurkiran.harikrishnan@xilinx.com>
Date: Fri, 11 Dec 2020 16:21:38 -0800
Subject: [PATCH] libweston: Remove substitute format for ARGB8888

Xilinx DP gfx layer does not support XRGB8888. Hence, remove the same
as opaque substitute.

Signed-off-by: Madhurkiran Harikrishnan <madhurkiran.harikrishnan@xilinx.com>
Upstream-Status: Inappropriate [Xilinx specific]

Reworked the patch for the newer weston

Signed-off-by: Mark Hatle <mark.hatle@amd.com>
---
 libweston/pixel-formats.c | 1 -
 1 file changed, 1 deletion(-)

Index: weston-13.0.0/libweston/pixel-formats.c
===================================================================
--- weston-13.0.0.orig/libweston/pixel-formats.c
+++ weston-13.0.0/libweston/pixel-formats.c
@@ -251,7 +251,6 @@ static const struct pixel_format_info pi
 	{
 		DRM_FORMAT(ARGB8888),
 		BITS_RGBA_FIXED(8, 8, 8, 8),
-		.opaque_substitute = DRM_FORMAT_XRGB8888,
 		.addfb_legacy_depth = 32,
 		.bpp = 32,
 		GL_INTERNALFORMAT(GL_RGBA8),
